src/components/routes/RoutePage.jsx

```diff
@@ -46,7 +54,19 @@ function RoutePage() {
           <h1 className="text-4xl font-extrabold z-50 relative word-break">{title}</h1>
         </div>
         <div className="flex flex-col items-end">
-          <div className="w-3/5 mb-10 text-end">Tryk på kortet for at undersøge ruten</div>
+          {openStreetMapConsent && <div className="w-3/5 mb-10 text-end">Tryk på kortet for at undersøge ruten</div>}
+
+          {!openStreetMapConsent && !focusOnText && (
+            <button type="button" onClick={() => setFocusOnText(true)} className="opacity-35 mb-10 text-center">
+              Du har ikke givet samtykke, derfor kan vi ikke vise er kort her. Vil du ændre det, kan du trykke her
+            </button>
+          )}
+          {!openStreetMapConsent && focusOnText && (
+            <button type="button" onClick={() => resetPermission()} className="mb-10 text-center">
+              Du har ikke givet samtykke, derfor kan vi ikke vise er kort her. Vil du ændre det, kan du trykke her
```

Comment:
We have the exact same message twice in the code. Consider putting it into a variable.